src/tests/test_files/dynamo0p3/12.4_enforce_op_bc_kernel.f90

```diff
@@ -1,9 +1,42 @@
+! -----------------------------------------------------------------------------
+! BSD 3-Clause License
+!
+! Copyright (c) 2017, Science and Technology Facilities Council
+! However, it has been created with the help of the GungHo Consortium,
```

Comment:
We don't need the GungHo bit as this was created after GungHo finished. If it wasn't we need the original GungHo license here instead (until we get agreement that we can take ownership).